frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeTableRow.tsx

```diff
@@ -30,7 +30,13 @@ const CustomServingRuntimeTableRow: React.FC<CustomServingRuntimeTableRowProps>
   const templateOOTB = isTemplateOOTB(template);
 
   return (
-    <Tr key={rowIndex} id={servingRuntimeName} draggable {...props}>
+    <Tr
+      key={rowIndex}
+      id={servingRuntimeName}
+      data-testid={servingRuntimeName}
```

Comment:
Assign a static test ID as well as dynamic once https://github.com/opendatahub-io/odh-dashboard/pull/2579 is fixed.